### PR TITLE
#5209 activity tab

### LINKED ISSUE #5209: Activity tab improvements
image
- we are changing the buttons so it's easier to spot what button is active 
- updated gallery item detail file: https://www.figma.com/file/gHSzqGPeQ9qK4L0qIDcTjY/gallery-item-handoff?node-id=1270%3A5615&t=GFYf3JbvVHzSvhDH-1

button divided into: default, hover, active and active hover

can we also remake the spacing in the activity tab so the date is not jumping to the second row - on desktop mainly 
image
- maybe lower the spacing between event and price

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #5209
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-03-09 at 7 49 30 PM

### CODE REVIEW

File: components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue [hunk 1]
```diff
@@ -79,7 +82,8 @@ const cssActive = (value) => {
   }
 
   &-filter {
-    column-gap: 2rem;
+    column-gap: 1.5rem;
```
Comment (daiagi) [inline components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue:85]: I think the columm-gap is too large compare with the design
image

File: components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue [hunk 2]
```diff
@@ -1,21 +1,24 @@
 <template>
   <div class="gallery-activity-events-wrapper is-flex is-flex-direction-column">
-    <div class="events p-5 is-flex is-flex-direction-column">
+    <div class="events px-5 py-4 is-flex is-flex-direction-column">
       <div class="events-filter is-flex is-flex-wrap-wrap">
-        <div class="events-checkbox" @click="checkAll">
+        <div
+          class="events-checkbox is-flex is-align-items-center"
```
Comment (daiagi) [inline components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue:6]:
```suggestion
class="is-clickable is-capitalized is-flex is-align-items-center"
```

same goes for other uses of events-checkbox class

File: components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue [hunk 3]
```diff
@@ -70,7 +70,9 @@
         field="timestamp"
         :label="$t('tabs.tabActivity.date')">
         <o-tooltip :label="parseDate(props.row.timestamp)" position="left">
-          {{ formatToNow(props.row.timestamp) }}
+          <span class="date-column">{{
```
Comment (daiagi) [inline components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue:73]:
```suggestion
<span class="no-wrap">
```

File: components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue [hunk 4]
```diff
@@ -139,4 +141,8 @@ const formatPrice = (price) => {
 .gallery-item-activity-table {
   overflow-y: auto;
 }
+
+.date-column {
+  white-space: nowrap;
+}
```
Comment (daiagi) [inline components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue:147]: can be removed and uses replaced by class="no-wrap"

**Review by daiagi — CHANGES_REQUESTED** [review]
minor changes

overall wfm

**Comment by exezbcz** [comment]
the button does not work on the full width and height - please make the select are the whole button. 

image

if the name is too long we can I think cut it so its not jumping to the second row - that is the ideal situation that no content is jumping to second row 

can we also increase the vertical padding? something like 20 is good - between the buttons and the stroke. 
image

**Comment by prachi00** [comment]
Left:

- [ ] if the name is too long we can I think cut it so its not jumping to the second row (somewhere on the edge with the second name

can this be done in another issue? bit tricky to change the class in oruga table as this is deeply nested and would be through unscoped css

File: components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue [hunk 5]
```diff
@@ -1,24 +1,27 @@
...
+          class="is-clickable is-capitalized events-checkbox-container"
+          :class="cssActive(value)">
           <input
             :id="name"
             v-model="interactions"
             type="checkbox"
             :value="value"
             class="is-hidden" />
-          <label :for="name" :class="cssActive(value)">
+          <span :for="name" class="is-clickable">
```
Comment (daiagi) [inline components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue:21]:
```suggestion
          <span :for="name">
```

it works without it:

image

**Comment by prachi00** [comment]
> 

fixed for div, for label , it seems is-clickable is mandatory as even if we wrap it in a , the label tag inside it still wont show pointer without is-clickable

**Comment by exezbcz** [comment]
good! works for me

File: components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue [hunk 6]
```diff
@@ -1,24 +1,28 @@
 <template>
   <div class="gallery-activity-events-wrapper is-flex is-flex-direction-column">
-    <div class="events p-5 is-flex is-flex-direction-column">
+    <div class="events px-5 py-5 is-flex is-flex-direction-column">
```
Comment (roiLeo) [inline components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue:3]:
```suggestion
    <div class="events p-5 is-flex is-flex-direction-column">
```

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 25 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $25 @ 35.44 USD/KSM ~ 0.705 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xa86e1cab2a14ff1fed6612c4a0e0d059faec44cc8b3287c5a35a15f95e9db82f](https://kusama.subscan.io/extrinsic/0xa86e1cab2a14ff1fed6612c4a0e0d059faec44cc8b3287c5a35a15f95e9db82f)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**